src/Chapter4.hs

```diff
@@ -610,7 +629,10 @@ Implement the 'Monad' instance for our lists.
 🕯 HINT: You probably will need to implement a helper function (or
   maybe a few) to flatten lists of lists to a single list.
 -}
-
+instance Monad List where
+  (>>=) :: List a -> (a -> List b) -> List b
+  (>>=) Empty _ = Empty
+  (>>=) (Cons x xs) f = appendLists (f x) (xs >>= f)
```

Comment:
This implementation is correct :+1: You can either pattern match manually and reuse `appendLists`, or you can implement the `flatten :: List (List a) -> List a` function and reuse `Functor` instance for `List`. Either way is fine, and they both have the same performance :slightly_smiling_face: